GPU/GPUTracking/Base/GPUSettingsList.h

```diff
@@ -78,6 +78,10 @@ AddOptionRTC(retryRefit, char, 1, "", 0, "Retry refit when fit fails")
 AddOptionRTC(loopInterpolationInExtraPass, char, -1, "", 0, "Perform loop interpolation in an extra pass")
 AddOptionRTC(mergerReadFromTrackerDirectly, char, 1, "", 0, "Forward data directly from tracker to merger on GPU")
 AddOptionRTC(useMatLUT, char, 0, "", 0, "Use material lookup table for TPC refit")
+AddOptionRTC(trdMinTrackPt, float, .5f, "", 0, "Min Pt for tracks to be propagated through the TRD")
```

Comment:
Just one minor optimization: could you inside the rec structure reorder the parameters such that first you have all 32bit ones (like int and float) and only afterwards the 1bit ones (char / unsigned char) to avoid padding and minimize the size of the struct.
For the processing struct it is not so important since it doesn't reside on the GPU, but actually if you want you can also do it there. Won't hurt :)

Reply:
Sure, I will reorder